resources/views/register.blade.php

```diff
@@ -0,0 +1,34 @@
+@extends('layouts.app')
+@section('content')
+    <form action="{{ url("register") }}" method="POST">
+        @csrf
+        <div class="container">
+            <div class="panel panel-default px-4">
+                <div class="panel-heading">
+                    登録用メールアドレス
+                </div>
+                <div class="panel-body">
+                    <div class="form-group">
+                    @include('common.info')
+                    @empty($email)
+                        <label for="email-form">Eメールアドレス</label>
+                        <div>
+                            <input type="text" name="email" class="form-control">
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <div class="">
+                            <button>
+                                <i></i>送信
+                            </button>
+                        </div>
+                    </div>
+                    @endempty
+                    @isset($email)
+                    <p>{{$email}}宛にメールを送信しました。</p>
```

Comment:
```suggestion
                    <p>{{ $email }}宛にメールを送信しました。</p>
```
{{}} 内の前後にも半角スペース開けるようにしてください。

Reply:
Bladeファイル内の{{}}内のスペースを削除修正致しました。
ご確認よろしくお願い致します。
resources/views/common/info.blade.php
resources/views/emails/mailable.blade.php
resources/views/layouts/app.blade.php
resources/views/register.blade.php
resources/views/successful.blade.php
resources/views/create_user.blade.php
resources/views/tasks.blade.php